### ISSUE TITLE
Redesign file list ui

### ISSUE THREAD

**Issue by shhsu** [Microsoft/IIS.WebManager#371]
https://github.com/microsoft/IIS.WebManager/issues/271

**Comment by shhsu** [comment]
Keyboard navigation does not work. This includes arrow key for navigating and spacebar for selection

**Comment by shhsu** [comment]
File list ui are implemented in
file list
web file list
default document
log file
trace file

Ideally they should all be unified
